pkg/api/v1/types.go

```diff
@@ -1249,6 +1249,11 @@ type PodSpec struct {
 
 // PodSecurityContext holds pod-level security attributes and common container settings.
 type PodSecurityContext struct {
+	// SupplementalGroups can be used to specify a list of
+	// additional groups which the main container process will run
+	// as. This will be applied to all containers in the pod in
+	// addition to the primary group of the container.
+	SupplementalGroups []int `json:"supplementalGroups,omitempty"`
```

Comment:
Even better would be to define `type UnixGroupID int64` and `type UnixUserID int64`.  I'll file an issue